vtex-apps/b2b-organizations
# Feat/acl on b2b suite management message

#### What problem is this solving?
Seeks to notify the user of changes in the management of B2B organizations


On the admin/b2b-organizations/organizations/ and admin/b2b-organizations/organizations#/requests pages we will display a topbar informing users about upcoming changes

#### How to test it?
1. Go to admin/b2b-organizations/organizations/ and admin/b2b-organizations/organizations#/requests page and see the message in the topbar



[Workspace](https://josmarjr--b2bstore005.myvtex.com/admin/b2b-organizations/organizations/#/requests)

#### Screenshots or example usage:
<img width="1440" alt="Screenshot 2024-10-23 at 10 17 32" src="https://github.com/user-attachments/assets/0c29075c-1992-42eb-ad1d-6542f80617e0">




#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExamtkNzBtcXJrMHBmbGQxeDRobnVxYmc4cTZsYnRlOWIzZGk1bjd6MSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/xT5LMHxhOfscxPfIfm/giphy.gif)



![](put .gif link here - can be found under "advanced" on giphy)
